FBC/FunctionalEntities/Markets.rdf

```diff
@@ -334,6 +382,14 @@ OTFs are intended to be similar in scope to a swap execution facility (SEF), a t
 		<fibo-fnd-utl-av:explanatoryNote>Unlike RMs and MTFs, operators of OTFs will have discretion as to how to execute orders, subject to pre-transparency and best execution obligations.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fbc-fct-mkt;QuoteDrivenMarket">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fct-mkt;Exchange"/>
+		<rdfs:label>quote-driven market</rdfs:label>
+		<skos:definition>exchange in which prices are determined from bid and ask quotations made by market makers, dealers, or specialists</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>In a quote-driven market, dealers fill orders from their own inventory or by matching them with other orders. Note that this differs from a typical market, which is order-driven rather than price-driven.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
should the last phrase not be "quote-driven"?

Reply:
fixed